tuna/miopen_tables.py

```diff
@@ -473,29 +472,28 @@ class ConvolutionGolden(BASE, GoldenMixin):
   """Golden table for convolution"""
   __tablename__ = "conv_golden"
   __table_args__ = (UniqueConstraint("golden_miopen_v",
-                                     "find_db",
                                      "config",
-                                     "session",
+                                     "solver_id",
                                      "arch",
                                      "num_cu",
                                      name="uq_idx"),)
 
-  find_db = Column(Integer, ForeignKey("conv_find_db.id"), nullable=False)
```

Comment:
If we keep this foreign key it will be easier to link golden entries with their source in the conv_find_db table. Unless we are planning to keep conv_golden entries longer than their counterpart.

side: We should rename conv_find_db to conv_perf_db soon. Because that's what it is now.

Reply:
should we? thought we nuke the find_db every now and then? is our plan to just keep it there forever?